### DISCUSSION TITLE
cobbler PXE服务器，不安装dhcp，通过路由器DHCP功能自动获取IP是否能装系统

### DISCUSSION THREAD

**Discussion by xjr1** [cobbler/cobbler#2833] [Version specific questions]
cobbler PXE服务器，不安装dhcp，通过路由器DHCP功能自动获取IP是否能装系统，不然我的PXE服务器老是与我局域网其他设备冲突

**Comment by SchoolGuy** [comment]
Okay it appears Google Translate has reached its limit. Could you please edit the title and description so we can discuss this in English? It is the repository language after all. Sorry that I can help you in your native tongue but the list of languages I speak is limited to German an English sadly.

Edit: After I got your message translated by @chenti1019 : Then you need to choose subnets which are non conflicting or you need to make the configurations identical in a way that it doesn't matter which DHCP server is handing out the configuration. I believe this is a good example of how this can be achieved for example: https://serverfault.com/a/368517/408066 . The exact solution is dependent on the hardware used, the software used and the time & knowledge to architect such a setup. The easiest way to fix this would be to disable all DHCP servers expect one per managed network.

**Reply by chenti1019** [reply]
I happened to see the discussion, I'll help translate on this time.

He wants to use network devices such as switches and routers as dhcp server and distribute IP addresses.

And the dhcp server will cause IP conflicts with the existing private subnet.

**Reply by SchoolGuy** [reply]
Thanks!
